chainio/dispatcher.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	"github.com/btcsuite/btclog/v2"
 	"github.com/lightningnetwork/lnd/chainntnfs"
+	"golang.org/x/sync/errgroup"
```

Comment:
we might think of adding an error group with a context, so we fail as soon as we encounter the first error ?